app/views/admin/shared/_sidenav_product.html.haml

```diff
@@ -1,5 +1,9 @@
 %ul.sidebar-nav
-  = tab "Instruments (#{current_facility.instruments.count})", facility_instruments_path(current_facility), sidenav_tab == 'instruments'
-  = tab "Services (#{current_facility.services.count})", facility_services_path(current_facility), sidenav_tab == 'services'
-  = tab "Items (#{current_facility.items.count})", facility_items_path(current_facility), sidenav_tab == 'items'
-  = tab "Bundles (#{current_facility.bundles.count})", facility_bundles_path(current_facility), sidenav_tab == 'bundles'
+  = tab "#{Instrument.model_name.pluralize} (#{current_facility.instruments.count})", facility_instruments_path(current_facility), sidenav_tab == Instrument.model_name.plural
+  = tab "#{Service.model_name.pluralize} (#{current_facility.services.count})", facility_services_path(current_facility), sidenav_tab == Service.model_name.plural
+  = tab "#{Item.model_name.pluralize} (#{current_facility.items.count})", facility_items_path(current_facility), sidenav_tab == Item.model_name.plural
+  = tab "#{Bundle.model_name.pluralize} (#{current_facility.bundles.count})", facility_bundles_path(current_facility), sidenav_tab == Bundle.model_name.plural
+
+- if current_ability.can?(:index, TrainingRequest)
+  %ul.sidebar-nav
+    = tab "#{TrainingRequest.model_name.titleize.pluralize} (#{current_facility.training_requests.count})", facility_training_requests_path(current_facility) , sidenav_tab == TrainingRequest.model_name.plural
```

Comment:
do we need the `titleize`, or can we just set the model name to "Training Request" in the locales?